packages/swapper/src/utils.ts

```diff
@@ -332,3 +332,10 @@ export const checkSolanaSwapStatus = async ({
     return createDefaultStatusResponse(txHash)
   }
 }
+
+export const txStatusBySwapStatus: Record<SwapStatus, TxStatus> = {
```

Comment:
unused

Reply:
removed in [53df27f](https://github.com/shapeshift/web/pull/9695/commits/53df27f350df1a851f51947e6180239eb47782f0)